Toolset/palettes/standalone settings/revstandalonesettingsiosbehavior.livecodescript

```diff
@@ -379,12 +389,17 @@ function getProfiles
 end getProfiles
 
 on updateIosSettingState
-   set the enabled of group "iOS Settings" of me to (the hilite of button "Build for iOS" of me)
-   set the visible of button "iOSChecked" to (the hilite of button "Build for iOS" of me)
-   set the opaque of button "iOSChecked" to false
    -- SN-2015-03-30: [[ Bug 14794 ]] Add a checkbox for disable the 64-bit slice
    --  which is only valid for 5.1.1 built (for iPad 1).
    set the enabled of button "32bit_only" to (the hilite of button "Build for iOS")
+   set the enabled of button "iOSTabMenu" to (the hilite of button "Build for iOS")
+   set the disabled of group "Basic Settings" to (not the hilite of button "Build for iOS")
+   set the disabled of group "Requirements" to (not the hilite of button "Build for iOS")
+   set the disabled of group "Background Modes" to (not the hilite of button "Exits on Suspend")
```

Comment:
This should be `of group "Requirement"`, and it will fix the following bug:

1. Go to "Requirements" pane.
2. Uncheck the "Build for iOS"
Expected result: The group is disabled.
Observed result: It is not disabled